### ISSUE TITLE
在查询时存在琏式查询

### ISSUE THREAD

**Issue by iHDJ** [go-gorm/gorm#4065]
![image](https://user-images.githubusercontent.com/56471245/107116835-fc381a00-68b0-11eb-9edd-fd2b4ad7cc30.png)

图1中是第一次查询user.name = "alicex"的语句，最终结果也是查询成功的
![image](https://user-images.githubusercontent.com/56471245/107116862-1f62c980-68b1-11eb-8499-98c5b1588833.png)
图2中的第二次查询并不是查询user而是查询member，查询user_id = 1 and organization_id = 1。最终结果不成功。并且显示Unknown column 'user_id'

![image](https://user-images.githubusercontent.com/56471245/107116897-5b962a00-68b1-11eb-9471-fd4712c59e5b.png)
结果一看日志的时候，他的SQL语句是SELECT * FROM `users` WHERE name = 'alicex' AND (user_id = 1 and organization_id = 1)

**Comment by iHDJ** [comment]
@jinzhu

**Comment by daheige** [comment]
I have also encountered this bug. Before querying some sql in for, I defined the query object in advance outside the for loop, query=db.Table("user"); and then use err in for:= query.Where ("id in ?",ids).Find(&userInfo).Error;The sql parsed as a result is just select * from user where id in ("xxx") and id in("xxxx") and id in("xxx "); This is a known bug. I have had such a problem with this bug in the production environment; later, I will redefine the query every time in the for loop, and then the query will meet the expectations select * from user where id ind("xxx","yyy"); I think this bug, I hope gorm v2 can fix this problem and avoid some unexpected SQL execution online, which ultimately does not meet expectations.
